resources/quality/Infinity3D/Infinity3D_super_fine.inst.cfg

```diff
@@ -0,0 +1,66 @@
+[general]
+version = 2
+name = Super Fine
+definition = Infinity3d
+
+[metadata]
+setting_version = 20
+type = quality
+quality_type = Super Fine
+weight = 1
+global_quality = True
+
+[values]
+layer_height = 0.06
+layer_height_0 = 0.2
+initial_layer_line_width_factor = 100
+
+wall_thickness = 1.2
+wall_line_count = 3
+top_thickness = 0.72
+top_layers = 12
+bottom_thickness = 0.72
+bottom_layers = 12
+inset_direction = inside_out
+skin_outline_count = 0
+
+; infill_line_distance = 8
+infill_sparse_density = 15
+infill_sparse_thickness = 0.3
+skin_preshrink = 1.2
+top_skin_preshrink = 1.2
+bottom_skin_preshrink = 1.2
+expand_skins_expand_distance = 1.2
+top_skin_expand_distance = 1.2
+bottom_skin_expand_distance = 1.2
+
+speed_travel = 70
+speed_topbottom = 50
+speed_wall_x = 50
+speed_wall_0 = 50
+speed_wall = 50
+speed_infill = 60
+speed_print = 60
+speed_print_layer_0 = 40
+speed_travel_layer_0 = 50
+skirt_brim_speed = 40
+
+retraction_hop = 1
+retraction_hop_enabled = False
+
+magic_spiralize = False
+magic_mesh_surface_mode = normal
+
+adhesion_type = skirt
+skirt_line_count = 2
+brim_width = 5
+brim_line_count = 20
+raft_margin = 15
```

Comment:
```suggestion
```

Same settings everywhere. Please ensure that profiles only contain changes. This will make it a lot easier to maintain this in the future.